ui/actions/nuxeo-delete-blob-button.js

```diff
@@ -151,14 +165,41 @@ import '../nuxeo-button-styles.js';
     }
 
     _remove() {
-      this.$.operation.execute().then(() => {
-        this.dispatchEvent(
-          new CustomEvent('file-deleted', {
-            composed: true,
-            bubbles: true,
-          }),
-        );
-      });
+      const rowNo = this.xpath.split('/')[1];
+      this.batchId = this.document.properties['monschema:mesdonnees'][rowNo].fichier['upload-batch'];
+      this.fileId = this.document.properties['monschema:mesdonnees'][rowNo].fichier['upload-fileId'];
+      if (this.batchId && this.fileId) {
+        this.$.blobRequest.data = {};
+        this.$.blobRequest
+          .remove()
+          .then(() => {
+            this._deleteEvent();
+          })
+          .catch((error) => {
+            this.dispatchEvent(
+              new CustomEvent('error', {
+                bubbles: true,
+                composed: true,
+                detail: {
+                  error,
+                },
+              }),
+            );
+          });
+      } else {
+        this.$.operation.execute().then(() => {
+          this._deleteEvent();
+        });
+      }
+    }
+
+    _deleteEvent() {
```

Comment:
Kindly customize it(_deleteEvent() ) more and add parameters for name & Detail Object.

Reply:
updated